### DISCUSSION TITLE
[FORBIDDEN] You don't have permission to access this.

### DISCUSSION THREAD

**Discussion by biplast** [tractr/directus-sync#82] [Q&A]
Pull is working but when i Push it shows me

<img width="865" alt="CleanShot-Block Test · Directus-Google Chrome-2024-06-19 at 10 44 06@2x" src="https://github.com/tractr/directus-sync/assets/166382482/2d35795c-da5c-489f-9f5c-7cb51add6879">

**Comment by EdouardDem** [comment]
@biplast Can you share the output of `npx directus-sync push` and also the logs of Directus when you open this page ?
